lib/utils/utils.dart

```diff
@@ -44,23 +44,24 @@ StdFee calculateDefaultFee({
   );
 }
 
-/// Checks if the bytes length of a string [str]
-/// is less than a certain value [len]
+/// Returns [true] if the bytes length of a string [str] is less than a certain
+/// value [len].
 bool checkStringBytesLen(String str, int len) {
-  List<int> bytes = utf8.encode(str);
+  final bytes = utf8.encode(str);
   return bytes.length <= len;
 }
 
-/// Checks if a string [uuid] has a Uuid-v4 format
+/// Return [true] if the string [uuid] has a Uuid v4 format.
 bool matchUuidv4(String uuid) {
   final regExp = RegExp(
-    r"^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$",
+    r'^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$',
     caseSensitive: false,
   );
   return regExp.hasMatch(uuid);
 }
 
-/// Returns [true] if the provided [str] is a valid bech32 string.
+/// Returns [true] if the provided [str] is a valid bech32 string
+/// (no checksum check is done).
 /// See: https://en.bitcoin.it/wiki/Bech32
 bool matchBech32Format(String str) {
```

Comment:
This function doesn't match exactly bech32 format: some string like `----1---------------` can used.    
Anyway it's ok because the chain controls the format.     
Leave it and we will improve it before the version release.

Reply:
Thanks, I've missed that and many others, I suppose. When I started to work on the `docs` module I wanted to try porting the Python reference bech32 checksum, but it's not a trivial task (maybe with the Go implementation it's better) and left only a very basic regex as you see.